src/app/core/basic-datatypes/configurable-enum/configure-enum-popup/configure-enum-popup.component.html

```diff
@@ -24,7 +24,25 @@ <h1 matDialogTitle i18n="title of dropdown options popup dialog">
     </button>
   </mat-form-field>
 </mat-dialog-content>
+<div><mat-form-field
+    cdkDrag
+    class="full-width"
+    appearance="fill"
+  >
+    <fa-icon
+      icon="grip-vertical"
+      matIconPrefix
+      class="grab-icon margin-right-small"
+    ></fa-icon>
+    <button mat-icon-button matIconSuffix > <fa-icon icon="trash"></fa-icon>
+    </button>
+  </mat-form-field>
+  <mat-form-field>
+    <mat-label>Input</mat-label>
```

Comment:
@sadaf895 I think you accidentally included some changes from the previous topic, creating a new dropdown option. Can you undo these changes and push another commit with the previous version of these files, so that this only includes the relevant changes?